<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>345</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Fitsa Hats</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Poppins Medium</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fitsa Hats</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_store">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Register new store</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="statusTip">
         <string>Register new store</string>
        </property>
        <property name="text">
         <string>Store</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_menu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Register new menu</string>
        </property>
        <property name="statusTip">
         <string>Register new menu</string>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_transaction">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Make a transaction</string>
        </property>
        <property name="statusTip">
         <string>Make a transaction</string>
        </property>
        <property name="text">
         <string>Transaction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>View stores and menus</string>
        </property>
        <property name="statusTip">
         <string>View stores and menus</string>
        </property>
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_report">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Make a report</string>
        </property>
        <property name="statusTip">
         <string>Make a report</string>
        </property>
        <property name="text">
         <string>Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_help">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Help</string>
        </property>
        <property name="statusTip">
         <string>Help</string>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
